src/frontend/test_runner/src/lib.rs

```diff
@@ -145,55 +161,111 @@ impl TestCase {
         let frontend = LocalFrontend::new(FrontendOpts::default()).await;
         let session = frontend.session_ref();
 
-        let mut result = None;
-
         let placeholder_empty_vec = vec![];
 
+        // Since temp file will be deleted when it goes out of scope, so create source advance.
+        self.create_source_advance(session.clone()).await?;
```

Comment:
```suggestion
        self.create_source(session.clone()).await?;
```

Btw, "in advance" ,  not "advance"